src/mprismetadata.cpp

```diff
@@ -417,8 +426,9 @@ MprisMetaData::MprisMetaData(QObject *parent)
 
 QVariant MprisMetaData::trackId() const
 {
+    // Returns trackId in string format although for D-Bus we want it to look like an object path
```

Comment:
Oops. :bug:
Later it doesn't return `trackId` value but `priv->m_metaData[MetaFieldTrackId]` directly.

Reply:
Fixed.